packages/core/src/components/Dropdown/Dropdown.tsx

```diff
@@ -260,7 +260,7 @@ const Dropdown: VibeComponent<DropdownComponentProps, HTMLElement> & {
           : selectedOptions;
 
         if (customOnChange) {
-          customOnChange(newSelectedOptions, e);
+          customOnChange(newSelectedOptions.length > 0 ? newSelectedOptions : null, e);
```

Comment:
I'm not 100% this is needed here
wdyt?

Reply:
Can you elaborate on the change?

Reply:
I'm not sure newSelectedOptions could have been null at any scenario.
And if it cannot be null, it means our API is not consistent today, and can sometimes return null and sometimes empty array (?).

Reply:
So this change might be redundant? Or am I misunderstanding something?

Reply:
If it's not mandatory for the migration I wouldn't make changes here